.travis.yml

```diff
@@ -5,12 +5,13 @@ os: osx
 # Add a TEST_DIR=[directory] row for each library (or group of libraries) to test
 # Each will be run as an independent, hermetic test (in parallel when resources permit)
 # on Traivs.
+# TEST_TASKS specifies the Gradle tasks to run.
 env:
-  - TEST_DIR=com.google.code.gson-gson
-  - TEST_DIR=org.joda-joda-convert
-  - TEST_DIR=joda-time-joda-time
-  - TEST_DIR=org.joda-joda-primitives
-  - TEST_DIR=org.apache.commons-commons-lang3
+  - TEST_DIR=com.google.code.gson-gson TEST_TASKS=build
+  - TEST_DIR=org.joda-joda-convert TEST_TASKS=build
+  - TEST_DIR=joda-time-joda-time TEST_TASKS=build
+  - TEST_DIR=org.joda-joda-primitives TEST_TASKS=build
+  - TEST_DIR=org.apache.commons-commons-lang3 TEST_TASKS=build
```

Comment:
Since this is repeated, it's boiler plate that can be removed ;-)
For now assume that all libraries run the build task for testing. I think that's a good assumption anyway.

If it needs something else in the future, then enumerate the exceptions, not the standard way of operating it.